conftest.py

```diff
@@ -156,6 +182,42 @@ def task_1(employee_1, manager_1):
     return task
 
 
+@pytest.fixture
+def task_2(employee_1, manager_1):
+    task = Task.objects.create(title="new house",
+                               assignee=employee_1,
+                               created_by=manager_1,
+                               priority=Priority.HIGH,
+                               status=Status.BACKLOG,
+                               description="build me a house")
+    return task
+
+
+@pytest.fixture
+def task_2_data(employee_1, manager_1):
```

Comment:
@aradz1997 Can you explain why you need both `task_2` and `task_2_data`?
If you really need it, why don't you use `Priority.HIGH` and `Status.BACKLOG` here too?

Reply:
I tried to use it but it didn't work because it is excepted a string, same with if I use task_2.priority the test isn't working

Reply:
Casting to string is an option? For example `str(Status.BACKLOG)`

Reply:
@natifridman 
can't do that to the fixture object because when creating an object it expected the ENUM and not the string, but in the test, I need the string, so that's why I added the data